linera-base/src/crypto/ed25519.rs

```diff
@@ -2,6 +2,8 @@
 // Copyright (c) Zefchain Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+//! Defines ed25519 signature primitives used by the Linera protocol.
```

Comment:
```suggestion
//! Defines Ed25519 signature primitives used by the Linera protocol.
```

(It's usually capitalized; refers to the name Edwards. _Unlike_ secp256k1 which seems to be spelled lowercase usually.)